### PR TITLE
Bugfixes CCFRI 4209 4173 4095 3850

### PR DESCRIPTION
**Pull request by jenbeckett**
# Description
bugs 
https://eccbc.atlassian.net/browse/CCFRI-4209
https://eccbc.atlassian.net/browse/CCFRI-4173
https://eccbc.atlassian.net/browse/CCFRI-4095

and coming in shortly 
https://eccbc.atlassian.net/browse/CCFRI-3850

## Checklist

- [ ] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [ ] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: frontend/src/components/SummaryDeclaration.vue [hunk 1]
```diff
@@ -210,16 +210,6 @@
...
-                        :rfi-app="facility?.rfiApp"
-                        :ccfri-id="facility?.ccfri?.ccfriId"
-                        :facility-id="facility.facilityId"
-                        :change-rec-guid="facility?.changeRequestId"
-                        :program-year-id="summaryModel?.application?.programYearId"
-                        @is-summary-valid="isFormComplete"
-                      />
-                    </v-expansion-panel>
                     <v-expansion-panel v-if="facility?.nmfApp" variant="accordion" value="nmf-summary">
                       <NMFSummary
```
Comment (vietle-cgi) [inline frontend/src/components/SummaryDeclaration.vue:214]: I think NMF is also duplicate?

File: backend/src/components/application.js [hunk 2]
```diff
@@ -178,7 +178,7 @@ async function upsertParentFees(req, res) {
         ccof_frequency: feeGroup.feeFrequency,
         'ccof_ChildcareCategory@odata.bind': childCareCategory,
         'ccof_ProgramYear@odata.bind': programYear,
-        'ccof_ApplicationCCFRI@odata.bind': `/ccof_applicationccfris(${feeGroup.ccfriApplicationGuid})`,
+        //'ccof_ApplicationCCFRI@odata.bind': `/ccof_applicationccfris(${feeGroup.ccfriApplicationGuid})`,
```
Comment (vietle-cgi) [inline backend/src/components/application.js:181]: is it for the PROD issue fix? I thought you want to keep it?

**Review by vietle-cgi — CHANGES_REQUESTED** [review]

File: frontend/src/components/SummaryDeclaration.vue (hunk 1 above)
Comment (jenbeckett) [reply frontend/src/components/SummaryDeclaration.vue:214]: removed

File: backend/src/components/application.js (hunk 2 above)
Comment (jenbeckett) [reply backend/src/components/application.js:181]: yes!! good catch - we def want to keep it.

**Review by vietle-cgi — APPROVED** [review]

**Pull request merged**